tokenserver-common/src/error.rs

```diff
@@ -1,12 +1,12 @@
-use std::{cmp::PartialEq, error::Error, fmt};
-
 use actix_web::{HttpResponse, ResponseError};
 use backtrace::Backtrace;
 use http::StatusCode;
+use pyo3::prelude::PyErr;
 use serde::{
     ser::{SerializeMap, Serializer},
     Serialize,
 };
+use std::{cmp::PartialEq, error::Error, fmt};
```

Comment:
nit: Was this an IDE automatic rearrange? Let's put it back.

FYI (and I just noticed this, so I'm happy to write it down somewhere) `rustfmt` nightly [has a group_imports feature](https://rust-lang.github.io/rustfmt/?version=v1.8.0&search=#group_imports). I don't know if it's stabilizing any time soon but I'm looking forward to using its `StdExternalCrate` setting as it's the format we've attempted to adopt.

(it'd be nice if it even grouped separate crates that live in the same workspace into their own block..)

Reply:
I'll check out that feature, that would really be handy for sure!